server/cmwell-cons/src/main/scala/ctl.scala

```diff
@@ -315,9 +316,9 @@ abstract class Host(user: String,
     // scalastyle:on
   }
 
-  def ips = ipMappings.getIps
+  def ips = hostIps.toList
 
-  def getSize = size
+  def getSize = esSize
```

Comment:
remove `getSize` method; each usage should use `esSize` member instead.

Reply:
can't do it since Grid and GridSubdiv use it